packages/launchpad/cypress/e2e/support/dropFileWithPath.ts

```diff
@@ -0,0 +1,29 @@
+import path from 'path'
+
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      dropFileWithPath(filePath: string): Cypress.Chainable<JQuery<HTMLInputElement>>
```

Comment:
@tgriesser I'm pretty sure my changes to pass the typescript linter aren't 100% correct...If I add types to the actual dropFileWithPath, I get errors and/or if I update the interface to accept the subject, my usage within the tests are not valid. Am I missing something obvious? Should I be extending [this](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts#L33)?

Reply:
@emilyrohrbough Hmm, I ran into this same issue recently somewhere else and gave up after trying to fix it for 5-10 minutes. 

I think what you have here is probably correct though, and it's somewhat confirmed by this guide: https://github.com/cypress-io/cypress-documentation/blob/master/content/guides/tooling/typescript-support.md#adding-child-or-dual-commands